.github/workflows/ci-llama-large-tests.yaml

```diff
@@ -68,8 +68,8 @@ jobs:
 
           # Test with pinned nightly releases, not what iree-turbine uses.
           pip install -f https://iree.dev/pip-release-links.html --upgrade \
-            iree-base-compiler==2.9.0rc20241107 \
-            iree-base-runtime==2.9.0rc20241107 \
+            iree-base-compiler \
+            iree-base-runtime \
```

Comment:
Keep comments up to date when changing code.

You can copy the text from other workflows that leave the version unpinned:
https://github.com/nod-ai/SHARK-Platform/blob/27b061e376276db1799f96360ca68176c4692642/.github/workflows/ci-shark-platform.yml#L65-L66

Reply:
I just bumped to the latest nightly, thus this shouldn't be required (for now).